src/test/java/com/johnstarich/moviematcher/controllers/MovieServiceTest.java

```diff
@@ -0,0 +1,101 @@
+package com.johnstarich.moviematcher.controllers;
+
+import com.google.gson.internal.LinkedTreeMap;
+import com.johnstarich.moviematcher.models.Movie;
+import com.johnstarich.moviematcher.models.Session;
+import com.johnstarich.moviematcher.models.User;
+import org.bson.types.ObjectId;
+import spark.Spark;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.List;
+
+/**
+ * Test for MovieService
+ * Created by gonzalezcesar on 7/16/16.
+ */
+public class MovieServiceTest extends AbstractMongoDBTest {
+	private final MovieService movieService = new MovieService();
+	private final UserService userService = new UserService();
+
+	@Override
+	public void setUp() throws Exception {
+		super.setUp();
+		userService.init();
+		movieService.init();
+		Spark.awaitInitialization();
+	}
+
+	public User register(String password) throws Exception {
+		return new User(null, "Test0", "foo", "bar").register(password);
+	}
+
+	public Session login(String password) throws Exception {
+		return User.login("Test0", password);
+	}
+
+	public Map<String, String> authHeaders(Session session) {
+		if(session == null) throw new AssertionError("Session is null");
+		if(session.id == null) throw new AssertionError("Session ID is null");
+		return Collections.singletonMap("Authorization", session.id.toHexString());
+	}
+
+	public Movie addMovie(String title) throws Exception {
+		return new Movie(new ObjectId(), title, null, null, null, null, null, null, null)
+			.save();
+	}
+    public Movie addMovie(ObjectId id, String title) throws Exception {
+        return new Movie(id, title, null, null, null, null, null, null, null)
+                .save();
+    }
```

Comment:
Something seems off about the indentation on this one. Looks like it was written with spaces instead of tabs.